package.json

```
@@ -5,8 +5,8 @@
   "dependencies": {
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
-    "@mui/icons-material": "^5.8.4",
-    "@mui/material": "^5.8.5",
+    "@mui/icons-material": "^5.8.6",
```

```suggestion
    "@mui/icons-material": "^5.8.4",
```
There are no version of @mui/icons-material beyond 5.8.4